scripts/examples/gn_efr32_example.sh

```diff
@@ -26,11 +26,17 @@ set -x
 env
 
 if [ -z "$3" ]; then
-    gn gen --check --fail-on-unused-args --root="$1" "$2"/"$EFR32_BOARD"/
+    gn gen --check --fail-on-unused-args --root="$1" --args="chip_config_memory_management=\"platform\"" "$2"/"$EFR32_BOARD"/
     ninja -v -C "$2"/"$EFR32_BOARD"/
+    #output commands for IDE navigation
+    ninja -C "$2"/"$EFR32_BOARD"/ -t compdb objc objcxx alink solink asm cc cxx >.vscode/compile_commands.json
+    #print stats
     arm-none-eabi-size -A "$2"/"$EFR32_BOARD"/*.out
 else
-    gn gen --check --fail-on-unused-args --root="$1" --args="efr32_board=\"$3\"" "$2/$3"
+    gn gen --check --fail-on-unused-args --root="$1" --args="chip_config_memory_management=\"platform\" efr32_board=\"$3\"" "$2/$3"
     ninja -v -C "$2/$3"
+    #output commands for IDE navigation
+    ninja -C "$2/$3" -t compdb objc objcxx alink solink asm cc cxx >.vscode/compile_commands.json
```

Comment:
Can we not add this here?

IMO - This script shouldn't make changes outside the designated output directory.